AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Models/ReleaseNotesBuilder.swift

```diff
@@ -15,7 +15,8 @@ struct ReleaseNotesBuilder {
     let repoOrg: PrepareRelease.Org
     let repoType: PrepareRelease.Repo
     let commits: [String]
-    var featuresReader: FeaturesReader = FeaturesReader()
+    let features: Features
+    let featuresIDToServiceName: [String: String]
```

Comment:
ReleaseNotesBuilders takes the feature list & mapping as inputs rather than reading them itself.